time time
uint16 mode	# 0 : Auto Drive / 1 : Emergency Stop / 2 : Manual Drive
